api/admin/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## Changelog
```

Comment:
This looks good. Can you add one sentence to make it explicit that this changelog is different from the one at the root and that this only tracks changes to the web admin.

Reply:
Updated!